Source/CsDebugScript.Engine/Engine/Utility/DictionaryCache.cs

```diff
@@ -62,6 +67,22 @@ public int Count
         }
 
         /// <summary>
+        /// Gets the Cached values.
+        /// </summary>
+        IEnumerable ICacheCollection.ValuesRaw { get { return Values; } }
```

Comment:
Done.

---
In reply to: [70602055](https://github.com/southpolenator/WinDbgCs/pull/8#discussion_r70602055) [](ancestors = 70602055)